tracer/main.py

```diff
@@ -22,18 +23,28 @@
     is_flag=True,
     help="Print detailed report on each hop along the path",
 )
-def main(url, full):
+@click.option(
+    "--output",
+    "-o",
+    multiple=True,
+    help="File location where you want to save the Output",
+)
+def main(url, full,output):
+    
     """
     HTTP-Tracer returns the redirects on way to the destination URL.
 
     If user does not supply 'https://' it will default to 'http://'
     """
+    if(output):
+        sys.stdout = open(output[0], 'w')
```

Comment:
I can see what you're doing here but why is it `output[0]`. It's not immediately apparent what `output[0]` is both as a data structure and its implementation.

Is `output[0]` a string, or list?